src/OSWindow-Core/OSWorldRenderer.class.st

```diff
@@ -126,21 +128,25 @@ OSWorldRenderer >> displayWorldState: aWorldState ofWorld: aWorld submorphs: sub
 { #category : #initialization }
 OSWorldRenderer >> doActivate [
 
-	| attributes |
+	| attributes initialExtent |
+	
+	initialExtent := world worldState realWindowExtent ifNil: [1500@1000].
```

Comment:
Would be nice to define World method #realWindowExtent with default value being there instead of in the renderer.

Reply:
And maybe different name for world method like #preferredExtentForStartup to be more explicit that it is not an actual current window extent